docs/doxygen/ie_docs.xml

```diff
@@ -338,17 +339,6 @@ limitations under the License.
                 <tab type="user" title="Glossary" url="@ref openvino_docs_IE_DG_Glossary"/>
             </tab>
 
-            <!-- nGraph -->
-            <tab type="usergroup" title="nGraph Developer Guide" url="@ref openvino_docs_nGraph_DG_DevGuide">
-                <tab type="user" title="Introduction" url="@ref openvino_docs_nGraph_DG_Introduction"/>
-                <tab type="user" title="Basic Concepts" url="@ref openvino_docs_nGraph_DG_basic_concepts"/>
-                <tab type="user" title="Operation sets" url="@ref openvino_docs_MO_DG_IR_and_opsets"/>
-                <tab type="user" title="Graph building" url="@ref openvino_docs_nGraph_DG_build_function"/>
-                <tab type="user" title="Transfomations" url="@ref ngraph_transformation"/>
```

Comment:
looks like transformations are missed

Reply:
I added transformations to this file: https://github.com/openvinotoolkit/openvino/pull/10055/commits/d938faca49d73faef3a03e89ee07d7b3809f0888#diff-a977e3f4c1d4c3b03d3875d19df4c4b141f45a65688a3b5cbebe11b34ac66987R12

Reply:
@ilyachur  Please remove this obsolete . xml file